muxtools/muxing/tracks.py

```diff
@@ -60,13 +60,13 @@ def __init__(
     def mkvmerge_args(self) -> list[str]:
         filepath = str(self.file.resolve())
         if self.type == TrackType.ATTACHMENT:
-            is_font = self.file.suffix.lower() in [".ttf", ".otf"]
+            is_font = self.file.suffix.lower() in [".ttf", ".otf", ".ttc", ".woff", ".woff2"]
```

Comment:
You should also add ``.otc``.
These kind of font are really rare, but they do exist.
See microsoft doc: https://learn.microsoft.com/en-us/typography/opentype/spec/otff#filenames